integration-cli/docker_cli_run_test.go

```diff
@@ -2704,3 +2704,7 @@ func (s *DockerTrustSuite) TestTrustedRunFromBadTrustServer(c *check.C) {
 		c.Fatalf("Missing expected output on trusted push:\n%s", out)
 	}
 }
+
+func (s *DockerSuite) TestRunCapAddSYSTIME(c *check.C) {
+	dockerCmd(c, "run", "--cap-drop=ALL", "--cap-add=SYS_TIME", "busybox", "sh", "-c", "date --set `date '+%H:%M:%S'`")
```

Comment:
@sunyuan3 sorry to bother again, but is there a way you could just query the capabilities you have and check SYS_TIME is part of it?